spec/rails_helper.rb

```diff
@@ -8,6 +8,9 @@
 # Prevent database truncation if the environment is production
 abort("The Rails environment is running in production mode!") if Rails.env.production?
 require 'rspec/rails'
+require "simplecov"
+SimpleCov.start
```

Comment:
You are welcome to include SimpleCov but just know that it's not required for an IC!